docs/docs-content/clusters/edge/local-ui/host-management/reset-reboot.md

```diff
@@ -48,7 +48,8 @@ Edge host is rebooting.
 You can reset an airgapped Edge host to factory default from Local UI. This restores the Edge host to the state right
 after the [initial configuration](../../site-deployment/site-installation/initial-setup.md) is completed in the Terminal
 User Interface (TUI). Reset to factory default removes all workloads, content, and cluster definition from the Edge
```

Comment:
@lennessyy , this was a new change - https://spectrocloud.atlassian.net/browse/PE-5620
Edge Host 'Reset' will preserve the link status and TUI configuration(static IP, DNC, hostname).